lms/static/styles/components/_BookSelector.scss

```diff
@@ -0,0 +1,11 @@
+.BookSelector {
+  @media screen and (min-width: 42rem) {
```

Comment:
Where do these breakpoints come from?

Reply:
I'm about to push a fixup commit in which I expand the comment here to read:

```
  // The containing `Modal` component will take up 90vw on narrow screens.
  // For viewports that are wider, `Modal` sets a min-width, but it's a little
  // narrow for this particular interface. The book-selector feels about right
  // at 36em where space allows. 42em minimum here allows for at least 3em
  // of horizontal breathing room on each side of the modal's contents.
```

At a higher level, I believe the `Modal` (responsive) default width constraints to be generally appropriate for most cases. This is a case in which we want to do some tuning specific to this interface.

When I tinkered, I found that `36em` _felt_ about the best for the width, where we have room to do it. Because we still want to be sure that there is horizontal space around the modal, I chose `42em` as the media query's `min-width`. The underlying Modal styling will still apply dimensional and positioning rules other than this one override for wider screens.

Are these numbers too magical? I'm not sure how rigorous to be here. I do have a gut feeling that we shouldn't try to account for this interface's specific needs in the Modal's own styling. It feels appropriate to me to do this kind of fine-tuning override to the interface at hand. Thoughts?